src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -5,71 +5,69 @@
     <div *ngIf="!isTestRun" class="row my-3 d-flex justify-content-between">
-        <div class="col-sm-8">
-            <div class="row">
-                <div class="col-8">
-                    <p class="h3" style="text-align: left">
-                        {{ 'artemisApp.exerciseAssessmentDashboard.yourStatus' | artemisTranslate }}:
-                        {{ 'artemisApp.exerciseAssessmentDashboard.tutorParticipationStatus.title.' + yourStatusTitle | artemisTranslate }}
-                    </p>
-                    <jhi-tutor-participation-graph
-                        [exercise]="exercise"
-                        [tutorParticipation]="tutorParticipation"
-                        [numberOfSubmissions]="numberOfSubmissions"
-                        [totalNumberOfAssessments]="totalNumberOfAssessments"
-                        [numberOfComplaints]="complaintsDashboardInfo.total"
-                        [numberOfOpenComplaints]="complaintsDashboardInfo.total - complaintsDashboardInfo.done!"
-                        [numberOfMoreFeedbackRequests]="moreFeedbackRequestsDashboardInfo.total"
-                        [numberOfOpenMoreFeedbackRequests]="moreFeedbackRequestsDashboardInfo.total - moreFeedbackRequestsDashboardInfo.done!"
-                        [numberOfAssessmentsOfCorrectionRounds]="numberOfAssessmentsOfCorrectionRounds"
-                        style="display: block"
-                    >
-                    </jhi-tutor-participation-graph>
-                    <p style="text-align: left; float: left; font-size: 16px">
-                        {{ 'artemisApp.exerciseAssessmentDashboard.tutorParticipationStatus.description.' + tutorParticipation.status!.toString() | artemisTranslate }}
-                    </p>
-
-                    <div
-                        *ngIf="isAtLeastInstructor && exam?.numberOfCorrectionRoundsInExam && exam!.numberOfCorrectionRoundsInExam! > 1 && isExamMode && !isTestRun"
-                        style="display: inline-block"
-                    >
-                        <ng-container *jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_INSTRUCTOR']">
-                            <jhi-second-correction-enable-button
-                                (ngModelChange)="toggleSecondCorrection()"
-                                [secondCorrectionEnabled]="secondCorrectionEnabled"
-                                [togglingSecondCorrectionButton]="togglingSecondCorrectionButton"
-                            ></jhi-second-correction-enable-button>
-                            <a class="btn btn-primary btn-sm ms-1" [routerLink]="getSubmissionsLinkForExercise(exercise)">
-                                <span jhiTranslate="artemisApp.assessmentDashboard.submissions">Submissions</span>
-                            </a>
-                        </ng-container>
-                    </div>
-                </div>
+        <div class="col-12 col-md-7 col-xl-4">
+            <p class="h3" style="text-align: left">
```

Comment:
There is a scss file - i suggest to move it :+1: (or: Can you use bootstraps classes?)